code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2874,8 +2874,14 @@
 	ph = 4.6 // Ants contain Formic Acid
 	/// How much damage the ants are going to be doing (rises with each tick the ants are in someone's body)
 	var/ant_damage = 0
+	/// Multiplied by ant_damage to find the damage dealt
+	var/ant_damage_coeff = 0.025
```

Comment:
ditto here. also this comment is more like, how it's used currently rather then what it represents, which makes it at best redundant and at worse makes it a lie. comments like this should describe intent rather then the exact process

Reply:
Updated this and the other one, hopefully a bit better now